### DISCUSSION TITLE
Error when publishing to Play Console: Build failed :| Unable to read manifest from Android App Bundle

### DISCUSSION THREAD

**Discussion by robbe-tistaert** [codemagic-ci-cd/codemagic-docs#2528] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Ionic/Cordova

### Steps to reproduce

Via gradle I create an .apk and an .aab file and output them as artifacts.

### Expected results

How can I see more logs of the failed build during the publishing of the .aab?

### Actual results

In the following step I want to publish the .aab to play console, but I get the error: 
`Build failed :|  
Unable to read manifest from Android App Bundle`

### Build id (optional)

66448c42b9a98f11aa7e69aa

**Comment by dtrdic** [comment, marked as answer]
Hi @robbe-tistaert, I can see you are using a brew command in one of your build steps. Brew released [an update ](https://brew.sh/2024/05/14/homebrew-4.3.0/#:~:text=HOMEBREW_AUTOREMOVE%20is%20the%20default%20behaviour%20meaning%20that%20brew%20cleanup%20and%20brew%20uninstall%20automatically%20run%20brew%20autoremove)a few days ago, which by default runs a cleanup and removes some cached files, but in the process it removes some files from our build machine like openSSL library, so this could be the cause of your failed build.
​
To disable this behaviour try adding `HOMEBREW_NO_INSTALL_CLEANUP` variable before the brew command is invoked
```
​export HOMEBREW_NO_INSTALL_CLEANUP=1
brew install gradle
```
Also let me share an[ example ](https://github.com/codemagic-ci-cd/codemagic-sample-projects/blob/main/ionic/ionic-capacitor-demo-project/codemagic.yaml#L81)of the YAML configuration for Ionic apps to use as a reference. You can also follow [this guide](https://docs.codemagic.io/yaml-quick-start/building-an-ionic-app/), which will help you set up your yaml workflow
